myRobot/src/main/native/cpp/MyRobot.cpp

```diff
@@ -1,5 +1,5 @@
 /*----------------------------------------------------------------------------*/
-/* Copyright (c) 2016-2019 FIRST. All Rights Reserved.                        */
+/* Copyright (c) 2016-2020 FIRST. All Rights Reserved.                        */
```

Comment:
Revert this.  wpiformat will complain because it looks at the full commit history, but on merge this change will go away completely.